src/ert/scheduler/scheduler.py

```diff
@@ -318,19 +240,10 @@ async def _monitor_and_handle_tasks(
                         raise task_exception
 
             if not self.is_active():
-                if self._ee_uri is not None:
-                    try:
-                        await self._events.put(CLOSE_PUBLISHER_SENTINEL)
-                        await asyncio.wait_for(
-                            self._publisher_done.wait(), timeout=self._queue_timeout
-                        )
-                        await asyncio.wait_for(
-                            self._events.join(), timeout=self._queue_timeout
-                        )
-                    except asyncio.TimeoutError:
-                        logger.error(
-                            f"{self._events.qsize()} items left unprocessed in the queue!"
-                        )
+                if self._ensemble_evaluator_queue:
```

Comment:
is not None

Reply:
I've checked and queue, even created but empty will be fine with this if.

Reply:
for readability